packages/utils/src/lib/reports/formatting.unit.test.ts

```diff
@@ -126,100 +128,161 @@ describe('formatSourceLine', () => {
     [{ startLine: 2, endLine: 2 }, '2'],
     [{ startLine: 2, endLine: 3 }, '2-3'],
   ])('should format position %o as "%s"', (position, expected) => {
-    expect(
-      formatSourceLine({ file: '/packages/utils/src/index.ts', position }),
-    ).toBe(expected);
+    expect(formatSourceLine(position)).toBe(expected);
   });
 
-  it('should return an empty string when position is missing', () => {
-    expect(formatSourceLine({ file: '/packages/utils/src/index.ts' })).toBe('');
+  it('should return an empty string when the position is missing', () => {
+    expect(formatSourceLine(undefined)).toBe('');
   });
 });
 
 describe('linkToLocalSourceForIde', () => {
-  it('should not format file path as link when outputDir is undefined', () => {
+  it('should not format the file path as a link when outputDir is undefined (VS Code)', () => {
+    vi.stubEnv('TERM_PROGRAM', 'vscode');
+    vi.stubEnv('GITHUB_ACTIONS', 'false');
+
     expect(
       linkToLocalSourceForIde({
-        file: '/packages/utils/src/index.ts',
+        file: toUnixPath('/packages/utils/src/index.ts'),
       }).toString(),
     ).toBe('`/packages/utils/src/index.ts`');
   });
 
-  it('should format file path as link when outputDir is provided', () => {
+  it('should format the file path as a link when outputDir is defined (VS Code)', () => {
+    vi.stubEnv('TERM_PROGRAM', 'vscode');
+    vi.stubEnv('GITHUB_ACTIONS', 'false');
+    const filePath = toUnixPath('/packages/utils/src/index.ts');
+    const outputDir = toUnixPath('/.code-pushup');
```

Comment:
Please drop the leading slash, so that the test inputs use the expected format:

```suggestion
    const filePath = toUnixPath('packages/utils/src/index.ts');
    const outputDir = toUnixPath('.code-pushup');
```

Applies to multiple places in this file, and to `packages/utils/src/lib/reports/generate-md-report.unit.test.ts` as well.